mbed_lib.json

```diff
@@ -0,0 +1,50 @@
+{
+"name": "qspif",
+    "config": {
+        "QSPI_IO0": "NC",
```

Comment:
we can use QSPI FLASH pins that are currently under review, then this lib can be removed?